src/orquestra/quantum/operators/__init__.py

```diff
@@ -0,0 +1,21 @@
+################################################################################
+# © Copyright 2020-2022 Zapata Computing Inc.
+################################################################################
```

Comment:
question: do we need to add openfermion's license here? since this module mostly consists of Zapata's code rather than an openfermion clone. however, openfermion's license is in the `_pauli_operators.py` so if we want it there, we should probably put it here for consistency.

Reply:
No need to have openfermion license here, as no code in this init file comes from openfermion :)